modules/plugin/geopkg/src/main/java/org/geotools/geopkg/GeoPkgDataStoreFactory.java

```diff
@@ -106,6 +109,20 @@ protected String getDriverClassName() {
         return "org.sqlite.JDBC";
     }
 
+    @Override
+    protected SQLDialect createSQLDialect(JDBCDataStore dataStore, Map<String, ?> params) {
```

Comment:
Please add a test in this module (and then one to show the app-schema integration is working)

Reply:
The agreement with @nmco and @fernandor777 to put the implementation first then to put integration and unit tests.

Reply:
@ydeliorman no PR is merged without and test.